routes/resources.js

```diff
@@ -10,27 +10,44 @@ var Schemas = require('../generators/schemas');
 
 module.exports = function (app, model, Implementation, integrator, opts) {
   var modelName = Implementation.getModelName(model);
-  var schema = Schemas.schemas[Implementation.getModelName(model)];
+  var schema = Schemas.schemas[modelName];
+
+  function addSmartFieldToRecord(record, smartFieldObject, modelName) {
+    var smartFieldValue;
+    if(smartFieldObject.get || smartFieldObject.value) {
+      if (smartFieldObject.value) {
+        logger.warn('DEPRECATION WARNING: In your ' + modelName +
+          ' Model, Smart Field value method is deprecated. ' +
+          'Please use get method instead. ');
```

Comment:
What do you think about my previous comment about the change you made to this log?